### DISCUSSION TITLE
I'm sure I specified the correct URL, but I get a 404 error.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#139327] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

This code includes HTML, CSS and JS.
Please try the link below. I get a 404. What should I do? If anyone knows, please let me know.
[https://github.com/samugarinoko/countdown/](https://github.com/samugarinoko/countdown/)

**Comment by ghost** [comment]
I use Google Translate.

**Reply by maliksaqibahmad** [reply]
I think there's an issue with your site on GitHub pages so I think you should deploy your site using other hosting services like vercel or netlify.

**Reply by ghost** [reply]
Thank You.

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/5f721a01-bb72-4def-97d1-0542cb399270)

**Comment by davevad93** [comment, marked as answer]
Hi @samugarinoko, 

The problem is that your files aren't in the root of your repo, `index.html` should be in the root; this is your structure:

`/countdown`
`└── /.github/workflows`
              `         ├── index.html`
              `         ├── script.js`
              `         └── style.css`


But the right structure should be this:

`/countdown`
`├── index.html`
`├── script.js`
`└── style.css`

So basically you have to put the contents of `.github/workflows` in the root of the repo and delete the folder, then go to your [repo settings](https://github.com/samugarinoko/countdown/settings/pages) and make sure your website is built and deployed with the following configuration:

<img width="571" alt="0" src="https://github.com/user-attachments/assets/89ad1981-31e5-4f25-97e7-0b369ca896c1">

**Reply by ghost** [reply]
Oh,
Thank You.

**Reply by davevad93** [reply]
You're welcome @samugarinoko 👍

**Reply by ghost** [reply]
👍

**Reply by ghost** [reply]
Wow! I did it!!! Thank you very much! I'm very grateful.

**Comment by abdessalems** [comment]
A 404 error means the URL doesn't exist or is incorrect. Check for typos, ensure you're logged in for private resources, confirm the branch or file exists, and verify the URL is case-sensitive

**Reply by ghost** [reply]
Thank You.

**Comment by ghost** [comment]
Thanks everyone, @davevad93 for making this happen. Thank you!

**Reply by davevad93** [reply]
Glad to help @samugarinoko , that's the standard process for all repos hosted on Pages, remember that! 👍
